lib/developer_portal/lib/liquid/drops/message.rb

```diff
@@ -7,7 +7,7 @@ class Message < Drops::Model
 
       desc "Returns the ID of the message."
       def id
-        @model.id
+        message.id
```

Comment:
Not quite sure if that's correct, maybe we need to keep MessageRecipient's ID here